client/src/reviews/ratingsAndReviews/AttributesSummary.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { ReviewsMetadataContext } from '../../contexts/Contexts';
+
+const AttributesSummary: React.FC = () => {
+  const { reviewsMetadata } = useContext(ReviewsMetadataContext) || {};
+
+  if (reviewsMetadata) {
+    const characteristics = [];
+    for (const char in reviewsMetadata.characteristics) {
+      characteristics.push(<div key={char}>
```

Comment:
It might be a little cleaner if all this JSX was moved to its own component, but it doesn't matter too much